source/_templates/about.html

```diff
@@ -2,7 +2,7 @@ <h1><a href="https://nix.dev">nix.dev</a></h1>
 
 <iframe src="https://ghbtns.com/github-btn.html?user=nix-dot-dev&repo=nix.dev&type=star&count=true&size=large" frameborder="0" scrolling="0" width="170" height="30" title="GitHub"></iframe>
 
-<p>An opinionated guide for developers getting things done using <a href="https://nixos.org">the Nix ecosystem</a>.</p>
+<p>Official documentation for the Nix platform.</p>
```

Comment:
Here's an alternative that wouldn't deviate from the original gist, and also avoids going down the naming rabbit hole.

```suggestion
<p>Official documentation for getting things done with Nix</p>
```

Reply:
The Nix platform is cool and all, and maybe the documentation is a largely _about_ the Nix platform, but this documentation is not _for_ the Nix platform, but rather _for_ getting your own things done (where learning is also one of the things I suppose; not just guides but also tutorials). So I like this suggestion better.